architecture/core_concepts/routes.adoc

```diff
@@ -236,13 +237,14 @@ $ oc set env dc/router HAPROXY_ROUTER_SYSLOG_ADDRESS=127.0.0.1 HAPROXY_ROUTER_LO
 |`*ROUTER_SYSLOG_ADDRESS*` |  | Address to send log messages. Disabled if empty.
 |`*ROUTER_TCP_BALANCE_SCHEME*` | source | Load-balancing strategy for multiple endpoints for pass-through routes. Available options are `source`, `roundrobin`, or `leastconn`.
 |`*ROUTER_LOAD_BALANCE_ALGORITHM*` | leastconn | Load-balancing strategy routes with multiple endpoints. Available options are `source`, `roundrobin`, and `leastconn`.
-//|`*ROUTE_FIELDS*` |  | A field selector to apply to routes to watch, empty means all.
+//|`*ROUTE_FIELDS*` |  | A field selector to apply to routes to watch, empty means all. (FUTURE: it does not have complete support we need in upstream/k8s.)
 |`*ROUTE_LABELS*` |  | A label selector to apply to the routes to watch, empty means all.
 |`*STATS_PASSWORD*` |  | The password needed to access router stats (if the router implementation supports it).
 |`*STATS_PORT*` |  | Port to expose statistics on (if the router implementation supports it).  If not set, stats are not exposed.
 |`*STATS_USERNAME*` |  | The user name needed to access router stats (if the router implementation supports it).
 |`*TEMPLATE_FILE*` | `/var/lib/haproxy/conf/custom/haproxy-config-custom.template` | The path to the HAproxy template file (in the image).
 |`*RELOAD_INTERVAL*` | 12s | The minimum frequency the router is allowed to reload to accept new changes.
+|`*ROUTER_DISABLE_NAMESPACE_OWNERSHIP_CHECK*` | "" | Set to "true" to relax the namespace ownership policy.
```

Comment:
Any reason to use double-quotes instead of leaving the column blank? The inconsistency is conspicuous and not explained that I see.